draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -227,10 +227,19 @@ Properties are represented by RRsets below the corresponding member node.
 ## The Group Property
 
 The property is defined by a TXT record in the sub-node labelled `group`.
+By using this property, the producer can signal the consumer(s) to treat some member zones within the catalog zone differently.
+The consumer MAY apply different configuration options when configuring the member zones, based on the group property value.
+The exact mapping of configuration on each group is left on the consumer's implementation and configuration.
+
+The producer MAY assign the group property to all, some, or none member zones within a catalog zone.
```

Comment:
`none` --> `no`. Simpler: `to any number of member zones`

Reply:
I'd like to leave it like this the emphasize what a consumer may encounter.
I changed "or non member zones" into "none of the member zones"

Reply:
Resolved in commit 05736f23